src/js/components/Layout/Breadcrumbs.jsx

```diff
@@ -1,11 +1,67 @@
-import React from 'react';
+/* eslint-disable react/no-array-index-key */
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { showModal } from '../../actions';
 
-const Breadcrumbs = () => (
-  <div className="bradcrumbs-container d-flex">
-    <span>
-      Breadcrumbs
-    </span>
-  </div>
-);
+class Breadcrumbs extends Component {
+  constructor(props) {
+    super(props);
 
-export default Breadcrumbs;
+    this.openModal = this.openModal.bind(this);
+  }
+
+  openModal() {
+    this.props.showModal();
+  }
+
+  render() {
+    const listItems = window.location.pathname.split('/');
+    const listItemFormatted = [];
+    _.forEach(listItems, (name) => {
+      // Put a space before each uppercase
+      let nameFormatted = name.replace(/([A-Z])/g, ' $1').trim();
+      // Put an uppercase at the beginning of each word
+      nameFormatted = nameFormatted.replace(/(^\w{1})|(\s{1}\w{1})/g, match => match.toUpperCase());
+      listItemFormatted.push(nameFormatted);
+    });
+    const listToReturn = listItemFormatted.map((name, id) =>
+      (name === 'Openboxes' || name === '' ? null : (
+        <a key={`item-${id}`} href={window.location.pathname.split('/', id + 1).join('/')} className="item-breadcrumbs">
+          {name}
+          <img className="item-breadcrumbs" alt="/" src="/openboxes/images/bc_separator.png" />
```

Comment:
`/openboxes` needs to be configurable since the app will eventually need to be deployed under any context path.

Reply:
Same as answered above